### PR TITLE
Implementing New NMs

### PR DESCRIPTION
**Pull request by Caladius**
Added:
- Kegpaunch Doshgnosh
- Edacious Opo-opo
- QM1 for Edacious Opo-opo

Added .DS_Store to GitIgnore and removed from future commits.

### CODE REVIEW

File: sql/mob_pools.sql [hunk 1]
```diff
@@ -5861,6 +5861,7 @@ INSERT INTO `mob_pools` VALUES (5841,'Hovering_Hotpot','Hovering_Hotpot',175,0x0
 INSERT INTO `mob_pools` VALUES (5842,'Nargun','Nargun',135,0x0000B00100000000000000000000000000000000,1,1,11,320,100,0,1,0,0,2,0,32,0,0,0,0,0,1,0,1005);
 INSERT INTO `mob_pools` VALUES (5843,'Ignamoth','Ignamoth',253,0x0000100700000000000000000000000000000000,1,1,7,240,100,0,1,1,1,2,39,0,311,131,0,0,0,1,0,1006);
 INSERT INTO `mob_pools` VALUES (5844,'Ankabut','Ankabut',235,0x0000360100000000000000000000000000000000,1,1,7,240,100,1,0,0,1,2,0,0,1,131,0,0,0,1,0,1007);
+INSERT INTO `mob_pools` VALUES (5845,'Kegpaunch_Doshgnosh','Kegpaunch_Doshgnosh',190,0x0000AC0100000000000000000000000000000000,1,1,7,240,100,0,1,0,1,2,0,0,1868,1155,0,0,0,1,1,190);
```
Comment (nfaletra) [inline sql/mob_pools.sql]: Went and capped this boy real quick. entityFlags should be 155, not 1155. Surrounding mobs have flags values of 1155 so maybe that's where that came from.

**Comment by Caladius** [comment]
Unsure why is still says Changes Requested, unless im blind I addressed everything.

**Comment by TeoTwawki** [comment]
because that message doesn't disappear till he re-reviews the changes you made or his review gets dismissed

**Comment by Caladius** [comment]
ooooooooh

**Pull request merged**